### PR TITLE
Add user maintenance hooks

### LINKED ISSUE #3398: [Feature request] Add maintenance hooks
I think it could be useful to introduce new hooks for the extension system.
I am thinking hooks that can be use for maintenance purposes. It could be at the system level (perform a vacuum on sqlite databases), at the user level (remove useless configuration), or at the feed level (modify existing article content).

At the moment, we have the `feed_before_insert` hook which could be used for the feed level but it is triggered (as far as I understand) only when the feed is updated.
We also have the `freshrss_init` hook which could be used for the system level but it is trigger for every calls. It's probably too much for a maintenance task.

What I envisioned is hooks that are triggered only during the `actualize_script.php` process. In the file itself, we could have the hook at the system level and the hook at the user level. The hook at the feed level would have to be in the `feedController.php` process.

I suggest the following names:
- `freshrss_system_maintenance`
- `freshrss_user_maintenance`
- `freshrss_feed_maintenance`

### PR DESCRIPTION
**Pull request by aledeg**
Closes #3398 (partially)

Changes proposed in this pull request:

- Add user maintenance hooks

How to test the feature manually:

1. Add an extension using a new hook.
2. Run the `make refresh` command.

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested
- [ ] unit tests written (optional if too hard)
- [ ] documentation updated

[Additional information can be found in the documentation](https://github.com/FreshRSS/FreshRSS/tree/master/docs/en/developers/04_Pull_requests.md).

The new hook allows to add maintenance extensions at the user level.

### CODE REVIEW

File: app/actualize_script.php [hunk 1]
```diff
@@ -72,6 +72,9 @@ function notice($message) {
 	}
 
 	FreshRSS_Auth::giveAccess();
+
+	Minz_ExtensionManager::callHook('freshrss_user_maintenance');
```
Comment (Alkarex) [inline app/actualize_script.php:76]: Would be good to put it in the CLI version as well https://github.com/FreshRSS/FreshRSS/blob/master/cli/actualize-user.php

Comment (aledeg) [reply app/actualize_script.php:76]: I've tried but it seems that it is not triggered. :/

Comment (Alkarex) [reply app/actualize_script.php:76]: Could you please try with https://github.com/FreshRSS/FreshRSS/pull/3443 ?

Comment (aledeg) [reply app/actualize_script.php:76]: I've tried without success

Comment (Alkarex) [reply app/actualize_script.php:76]: Have you tested with a system or user extension?

Comment (Alkarex) [reply app/actualize_script.php:76]: Could you please try again with https://github.com/FreshRSS/FreshRSS/pull/3443 ?

Comment (aledeg) [reply app/actualize_script.php:76]: I tried with a system extension

Comment (aledeg) [reply app/actualize_script.php:76]: So far, no success either :/

Comment (Alkarex) [reply app/actualize_script.php:76]: Let's merge here and come back to it when https://github.com/FreshRSS/FreshRSS/pull/3443 works

**Pull request merged**